.env.sample

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL_TEST = postgresql://<username>:<password>@<host>/<databaseName>
```

Comment:
Kindly change the name of the file to `env_sample`.

Reply:
Thanks for the feedback. The changes have been implemented.